src/main/webapp/WEB-INF/views/orders/details.jsp

```diff
@@ -0,0 +1,43 @@
+<%@taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Order details</title>
+</head>
+<body>
+<table>
+    <tr>
+        <th><a href="${pageContext.request.contextPath}/users/registration">Registration</a></th>
+        <th><a href="${pageContext.request.contextPath}/users/all">User list</a></th>
+        <th><a href="${pageContext.request.contextPath}/products/add">Add product</a></th>
+        <th><a href="${pageContext.request.contextPath}/products/showproducts">Product list</a></th>
+        <th><a href="${pageContext.request.contextPath}/products/admin">Product list Admin</a></th>
+        <th><a href="${pageContext.request.contextPath}/carts">Shopping cart</a></th>
+        <th><a href="${pageContext.request.contextPath}/orders/user">Orders</a></th>
+        <th><a href="${pageContext.request.contextPath}/orders/admin">Orders Admin</a></th>
+        <th><a href="${pageContext.request.contextPath}/insertdata">Insert Data</a></th>
+    </tr>
```

Comment:
why do you repeat references to all pages in every page? Nothing critical but looks strange:)